protocols.go

```diff
@@ -26,6 +26,7 @@ const (
 	P_UDP   = 17
 	P_DCCP  = 33
 	P_IP6   = 41
+	P_QUIC  = 81
```

Comment:
81 SGTM, please also PR it into the table here: https://github.com/multiformats/multicodec/blob/master/table.csv